app/models/course_session.rb

```diff
@@ -20,10 +22,25 @@ def accepting_applications
   # Instance methods
   #
   def start_to_s
-    return self.start_date.strftime('%m-%d-%y')
+    return self.start_date.strftime('%m-%d-%y') if self.start_date
```

Comment:
You can also use the "safe navigation operator" here:

`self.start_date&.strftime('%m-%d-%y')`

Also `return` is not needed when something is the last statement in a method.